scripts/discordTravisPost

```diff
@@ -1,77 +0,0 @@
-#!/bin/bash
```

Comment:
Why was this deleted? I see it's still called from `.travis.yml`

Reply:
@jsantos17 It is obtained from `sbt-slamdata` via `transferCommonResources`.